.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Make a feature request if you have a suggestion for something new.
+title: ""
+labels: enhancement, triage
+assignees: ""
+---
+
+**Is your feature request related to a problem you're having? Please describe.**
+A clear and concise description of what the problem is. Such as: "I'm always frustrated when ..."
```

Comment:
"I'm always frustrated when ..." doesn't give us much information, and IMHO, makes this negative. We could just as easily have something like, "It would be great if I could ..." This also doesn't give information. Maybe we could give an example here that is rephrasing of an existing feature?

Reply:
I've removed the second sentence as it didn't add much.

Let's see how developers respond to this structure and then consider adding example issues references once we've collected enough that use this format.  These are min viable issue templates and we'll iterate.